configure.ac

```diff
@@ -1505,7 +1505,10 @@ AS_IF([test `uname -s` = "FreeBSD"],
 AS_IF([test -n "$with_compiler_plugins"],
       [CPPFLAGS="$CPPFLAGS -Xclang -load -Xclang ${with_compiler_plugins}/compilerplugins/obj/plugin.so -Xclang -add-plugin -Xclang loplugin -Xclang -plugin-arg-loplugin -Xclang --cool-base-path=\${abs_top_srcdir}"])
 
-COOLWSD_CONFIGDIR=${sysconfdir}/${PACKAGE}
+if test "x${prefix}" = "xNONE"; then
+    prefix=/usr/local
+fi
+eval COOLWSD_CONFIGDIR="${sysconfdir}/${PACKAGE}"
```

Comment:
Nit: why do you need eval here? I would expect sysconfdir is /etc and PACKAGE is coolwsd at this stage.

Reply:
Without eval the value was literal `${prefix}/etc/coolwsd` when I did not specify sysconfdir separately. The reason is that these paths are evaluated later. Autoconf sets it so, that prefix etc. can be changed even as late as `make install`.

Reply:
Ah I see, thanks.